freud/environment.pyx

```diff
@@ -532,11 +532,56 @@ cdef class EnvironmentCluster(_MatchEnv):
                 global_search=False):
         r"""Determine clusters of particles with matching environments.
 
-        In general, it is recommended to specify a number of neighbors rather
-        than just a distance cutoff as part of your neighbor querying when
-        performing this computation. Using a distance cutoff alone could easily
-        lead to situations where a point doesn't match a cluster because a
-        required neighbor is just outside the cutoff.
+        An environment is defined by the bond vectors between a particle and its
+        neighbors as defined by :code:`env_neighbors`.
+        For example, :code:`env_neighbors= {'num_neighbors': 8}` means that every
+        particle's local environment is defined by its 8 nearest neighbors.
+        Then, each particle's environment is compared to the environments of a
+        selected list of other particles in the system, controlled by the
+        :code:`neighbors` parameter. For example, :code:`neighbors={'r_max': 3.0}`
```

Comment:
Adding "each of" may lead to more confusion, because it could mean that we compute multiple environments for one particle, and we compare every single environment of one neighbor particle, which I don't think is what's happening.

Reply:
I added it for parallelism because you can't directly compare a particle's environment to a _list_ of particle environments. I meant for the "each of" to indicate the iteration that happens when comparing the environment to each element in the list of neighbor particle environments.

Reply:
> it could mean that we compute multiple environments for one particle, and we compare every single environment of one neighbor particle, which I don't think is what's happening.

Actually I do see the confusion here. I think the other half of my suggestion still helps clarity.

To resolve my confusion that led me to add "each of", we would need to rewrite the whole sentence.

I think this half of the change makes the paragraph more clear:

```suggestion
        Then, each particle's environment is compared to the environments of
        particles that satisfy a different cutoff parameter :code:`neighbors`.
        For example, :code:`neighbors={'r_max': 3.0}`
```